tools/istio-iptables/pkg/cmd/run.go

```diff
@@ -548,11 +548,32 @@ func (iptConfigurator *IptablesConfigurator) run() {
 		// save packet mark set by envoy.filters.listener.original_src as connection mark
 		iptConfigurator.iptables.AppendRuleV4(constants.PREROUTING, constants.MANGLE,
 			"-p", constants.TCP, "-m", "mark", "--mark", iptConfigurator.cfg.InboundTProxyMark, "-j", "CONNMARK", "--save-mark")
+		// If the packet is already marked with 1337, then return. This is to prevent mark envoy --> app traffic again.
```

Comment:
Constants.MARK is defined but not CONNMARK

Reply:
What do you mean here? Can you clarify a little bit?

Reply:
I guess what he meant was there is a constant defined for MARK but CONNMARK is hard coded

Reply:
Oh，we can add later